src/sentry/static/sentry/app/components/todos.jsx

```diff
@@ -0,0 +1,314 @@
+import React from 'react';
+
+import ApiMixin from '../mixins/apiMixin';
+import ConfigStore from '../stores/configStore';
+import OrganizationState from '../mixins/organizationState';
+
+const TodoItem = React.createClass({
+  mixins: [OrganizationState],
+
+  propTypes: {
+    task: React.PropTypes.object,
+  },
+
+  getInitialState: function() {
+    return {
+      showConfirmation: false
+    };
+  },
+
+  toggleConfirmation: function() {
+    this.setState({showConfirmation: !this.state.showConfirmation});
+  },
+
+  skip: function(task) {
+    this.props.onSkip(task);
+    this.setState({showConfirmation: false});
+  },
+
+  render: function() {
+    let org = this.getOrganization();
+
+    let classNames = '';
+    let description = '';
+    let doneDescription = "By being here, you've done it. Welcome to Sentry!";
+    if (this.props.task['status'] == 'Complete') {
+      classNames += ' checked';
+
+      if (this.props.task.task == 0) {
+        description = doneDescription;
+      } else {
+        description = this.props.task.user + ' completed this ' + moment(this.props.task.date_completed).fromNow();
+      }
+
+    } else if (this.props.task['status'] == 'Pending') {
```

Comment:
Using magical strings like this to hand off behavior is a bit scary to me. Since a simple typo will completely break this flow.

I'd prefer to see constants, like:  `const STATUS_PENDING = 'Pending'`, and then reference `this.props.task['status'] === STATUS_PENDING` instead.